webauthn/Sources/App/Controllers/WebAuthnController.swift

```diff
@@ -20,167 +20,126 @@ import HummingbirdCore
 import WebAuthn
 
 struct HBWebAuthnController {
+    let webauthn: WebAuthnManager
+
     func add(_ group: HBRouterGroup) {
```

Comment:
I know this isn't in scope for this PR. But I find this function extremely confusing to read, especially in `Application+configure.swift`. Can we add a label `to`, so it becomes:

```swift
func add(to group: HBRouterGroup)
```

.. and on the call site:

```swift
controller.add(to: self.router.group("api"))
```

Reply:
done